src/qml/DisplayPage.qml

```diff
@@ -105,6 +105,22 @@ Item {
                 }
```

Comment:
Doesnt the parent flickable's contentHeight need to be adjusted ? (for some reason github doesn't let me comment on that line but that would be l44

Reply:
Otherwise, feel free to merge once addressed.